source/examples/generated/FindOne.snippet.find-one-usage-example.kt

```diff
@@ -0,0 +1,30 @@
+import com.mongodb.client.model.Filters.lt
+import com.mongodb.client.model.Projections
+import com.mongodb.client.model.Sorts
+import com.mongodb.kotlin.client.coroutine.MongoClient
+import kotlinx.coroutines.flow.firstOrNull
+import kotlinx.coroutines.runBlocking
+
+data class Movie(val title: String, val runtime: Int, val imdb: IMDB) {
+    data class IMDB(val rating: Double)
+}
+
+fun main() = runBlocking {
+    // Replace the uri string with your MongoDB deployment's connection string
+    val uri = "<connection string uri>"
+    val mongoClient = MongoClient.create(uri)
+    val database = mongoClient.getDatabase("sample_mflix")
+    val collection = database.getCollection<Movie>("movies")
+
+    val projectionFields= Projections.fields(
+        Projections.include(Movie::title.name, Movie::imdb.name),
+        Projections.excludeId()
+    )
+    val resultsFlow = collection.find(lt(Movie::title.name, "The Room"))
+        .projection(projectionFields)
+        .sort(Sorts.descending("${Movie::imdb.name}.${Movie.IMDB::rating.name}"))
```

Comment:
The parameters for the Sort field look different here than in the FInd example (inclusion of $) -- do they both work? Maybe we should standardize

Reply:
They both work. The inclusion of $ is due to the dot notation with the property references to access the imdb.rating field